amplipi/streams.py

```diff
@@ -1331,7 +1348,15 @@ def _activate(self, vsrc: int):
           # NOTE: port 9000 is assumed
```

Comment:
This makes me feel like port should be used around here
I don't currently know where or how
I mean, it has to be appended like `lms_args += ['-s', server:port]`, but the use of "assumed" makes me feel that 9000 is input somewhere somehow automatically and I'll have to replace it first